package-builder/flox-build.mk

```
@@ -1,6 +1,6 @@
 #
 # This makefile implements Tom's stepladder from manifest to Nix builds:
-#
+
```

```suggestion
#
```
Minify diff; if making stylistic changes can do that in separate PR.